cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/EquipmentExportTest.java

```diff
@@ -199,145 +194,6 @@ void microGridCreateEquivalentInjectionAliases() throws IOException, XMLStreamEx
         assertTrue(compareNetworksEQdata(expected, actual));
     }
 
-    @Test
-    void nordic32() throws IOException, XMLStreamException {
```

Comment:
Shouldn't we also try to keep these tests?

(On a note side, there's still a CIM16 `nordic32.xml` file in the resources which doesn't seem to be called.)

Reply:
Which xiidm file could be used for the tests as a replacement for CIM14  nordic32.xiidm ?